src/node_jsvmapi.h

```diff
@@ -116,80 +116,99 @@ enum napi_valuetype {
   napi_function,
 };
 
-// Environment
-NODE_EXTERN napi_env napi_get_current_env();
+enum napi_errorvalue {
```

Comment:
The bit would allow you to avoid the call to napi_is_exception_pending in the case were it is useful to return a different error code.  

I don't think its all that hard to have some macros to mask out the bit/check if there was an exception.